homeassistant/scripts/benchmark/__init__.py

```diff
@@ -178,26 +179,28 @@ async def _logbook_filtering(hass, last_changed, last_updated):
 
     entity_attr_cache = logbook.EntityAttributeCache(hass)
 
-    def yield_events(event):
-        # pylint: disable=protected-access
-        entities_filter = logbook._generate_filter_from_config({})
+    entities_filter = convert_include_exclude_filter(
+        logbook.INCLUDE_EXCLUDE_BASE_FILTER_SCHEMA({})
+    )
+
+    def yield_events(event, entities_filter):
```

Comment:
```suggestion
    def yield_events(event):
```

Since this is nested you can drop `entities_filter` here

Reply:
Doesn't that apply to `event` as well? Its created earlier in the same method then passed in

Reply:
`event` changes in the loop but `entities_filter` doesn't so it should stay

Reply:
Ah gotcha, that was the criteria. Ok will change. Also will do the same change to the new benchmark (was passing in `entities_filter` and `config` as parameters)